snuba/utils/streams/batching.py

```diff
@@ -193,15 +168,13 @@ def _run_once(self) -> None:
         if self.producer:
             self.producer.poll(0.0)
 
-        msg = self.consumer.poll(timeout=1.0)
+        try:
```

Comment:
Now that the error management is done by the consumer, I think we may not need to have this method anymore and could just inline it into run(self)

Reply:
I agree, but right now this is used by the tests and I'd prefer not to introduce even more change here than is necessary. If we have some spare cycles later this is probably worth addressing.